client/app/components/template-explorer/template-explorer.component.js

```diff
@@ -0,0 +1,204 @@
+/* eslint camelcase: "off" */
```

Comment:
We are no longer actually using the Angular build in mechanism for actually loading the templates into `$templateCache`. That is actually happening in the webpack entrypoint via:

```js
// Angular templates
requireAll(require.context('./app', true, /\.html$/));
```

> Every single component we have uses this existing syntax.

The work I am finishing up this sprint involves moving lots of files around and therefore changing the relative import urls. Another advantage of this new syntax is that the import is relative only to the component that actually uses it and does not know anything else about our application structure.

I have done this conversion for all of the shared components in https://github.com/ManageIQ/manageiq-ui-service/pull/519